incubator/guides-restructured-text/src/RestructuredText/Parser/LinesIterator.php

```diff
@@ -28,18 +31,32 @@
 class LinesIterator implements Iterator
 {
     /** @var string[] */
-    private $lines = [];
-
-    /** @var int */
-    private $position = 0;
+    private array $lines = [];
+    private int $position = 0;
 
     public function load(string $document): void
     {
         $document = trim($this->prepareDocument($document));
         $this->lines = explode("\n", $document);
+
+        foreach ($this->lines as $key => $line) {
+            $this->lines[$key] = $this->tabToSpaces($line);
+        }
+
         $this->rewind();
     }
 
+    private function tabToSpaces(string $line): string
+    {
+        $tabStop = 8;
```

Comment:
Should this not be a setting? As people can mix their content using spaces and tabs the tabstop will make sense to configure. A tabstop of 2 will give a different results than a stop of 8

Reply:
As far as I was concerned, I wanted to keep it simple instead of adding just another setting :) Even if you mix content, it would not change the way the application works (unless you are inconsistent in your tab and space use, but I'm not sure a setting would help there ;) )